readthedocs/projects/models.py

```diff
@@ -850,6 +850,9 @@ def vcs_repo(
             )
         return repo
 
+    def get_vcs_class(self):
```

Comment:
This would keep the naming consistent w/ `vcs_repo()`
```suggestion
    def vcs_class(self):
```

Reply:
Also needs a docstring -- something like:

"""
This is used when we only need information about the VCS type, and not the actual on-disk repo.
"""